doc/engineering-reference/src/special-modules-reporting/heat-emissions.tex

```diff
@@ -0,0 +1,124 @@
+Q_{exh, zone} = m_{exh}(h_{zone} - h_{out})
+\end{equation}
+
+where \(m_{exf}\) is the exfiltration air flow rate, \(m_{exh}\) is the zone exhaust air flow rate from the zone exhaust node, \(h_{zone}\) and \(h_{out}\) are the zone and outdoor air enthalpy. Among these, the sensible parts are:
+
+\begin{equation}  \label{eq:he-4}
+Q_{exf, zone, sensible} = m_{exf} * c * (T_{zone} - T_{out})
+\end{equation}
+
+\begin{equation}  \label{eq:he-5}
+Q_{exh, zone, sensible} = m_{exh} * c * (T_{zone} - T_{out})
+\end{equation}
+
+where \(c)\ is the specific heat of air, \(T_{zone}\) and \(T_{out}\) are the zone and outdoor air temperature. And the latent parts are determined by taking the difference between the total and the sensible rate. For each zone, we calculate its exfiltration rate by solving a mass flow balance on the zone including infiltration, ventilation, outdoor air mixing, zone-to-zone mixing, and all of the zone inlet, exhaust, and return nodes.
+
+This part of zone exhaust only includes the exhaust heat at the zone level defined by \textbf{Fan:ZoneExhaust} or \textbf{AirflowNetwork:MultiZone:Component:ZoneExhaustFan}. For each zone, we aggregated this part by searching its exhaust fans. The exhaust heat from Zone HVAC system nodes is included in the next item. Besides, other part of air exchange with outdoors at system node level, balanced along with AirLoopHVAC, is considered in the next item.
+
+\subsection{Heat emission from HVAC systems through relief air}\label{emission-from-HVAC-relief}
+
+HVAC systems relieve heat from system outdoor air relief nodes, and the total and sensible emission rate are calculated as:
+
+\begin{equation}  \label{eq:he-6}
+Q_{exh, sys} = m_{exh}(h_{zone} - h_{out})
+\end{equation}
+
+\begin{equation}  \label{eq:he-7}
+Q_{exh, sys, sensible} = m_{exh} * c * (T_{zone} - T_{out})
+\end{equation}
+
+And the latent parts are determined by taking the difference between the total and sensible rate.
+
+The air exchange with outdoor is represented with an \textbf{OutdoorAir:Mixer} object in the \textbf{AirLoopHVAC:OutdoorAirSystem} object. We aggregate and report this part by each \textbf{Controller:OutdoorAir} object.
+
+\subsection{Heat rejection from HVAC system equipment}\label{emission-from-HVAC-reject}
+
+HVAC system equipment also directly exchanges heat with ambient air via refrigeration cycle and its energy usage. For example, for gas-fired boilers/heaters, waste heat is exhausted to outside together with combustion byproduct. For space cooling, heat is removed from indoors and dumped to outdoor at the condenser side or corresponding cooling equipment, such as cooling tower. In addition, energy consumed by outdoor equipment will directly dissipate to the ambient air, such as compressors and fans in packaged DX systems. In general, we assume the heat rejection is due to gas-powered combustion and condensers of any type, and we calculated these items based on the system type:
```

Comment:
Replace late sentence with:
In general, the heat rejection from gas-power combustion and condensers is calculated based on the system type: